docs/extensions/IconMarkupExtensions.md

```diff
@@ -0,0 +1,209 @@
+---
+title: FontIcon, FontIconSource, BitmapIcon markup extensions
+author: sergio0694
+description: The FontIcon, FontIconSource and BitmapIcon markup extensions allow developers to easily declare these types of icons directly from XAML in a compact manner.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, nullable bool, dependency property, markup extension, XAML, markup 
+---
+
+# IconMarkupExtensions
+
+The icon extensions are a group of markup extensions meant to simplify the creation of various icon types (specifically [`BitmapIcon`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon), [`BitmapIconSource`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIconSource), [`FontIcon`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.FontIcon) and [`FontIconSource`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.FontIconSource)) used across a variety of XAML controls. Using these extensions doesn't enable new capabilities per se, but it greatly simplifies the XAML syntax needed to create instances of these icon types. There are six such extensions available at the moment: `BitmapIconExtension`, `BitmapIconSourceExtension`, `FontIconExtension`, `FontIconSourceExtension`, `SymbolIconExtension` and `SymbolIconSourceExtension`.
```

Comment:
I know they don't return the object, but they're in the spirit of them, so should we still link to them? (if not up here, maybe in the related Topics at the bottom?

```suggestion
The icon extensions are a group of markup extensions meant to simplify the creation of various icon types (specifically [`BitmapIcon`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon), [`BitmapIconSource`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIconSource), [`FontIcon`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.FontIcon), [`FontIconSource`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.FontIconSource), [`SymbolIcon`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SymbolIcon), and [`SymbolIconSource`](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SymbolIconSource)) used across a variety of XAML controls. Using these extensions doesn't enable new capabilities per se, but it greatly simplifies the XAML syntax needed to create instances of these icon types. There are six such extensions available at the moment: `BitmapIconExtension`, `BitmapIconSourceExtension`, `FontIconExtension`, `FontIconSourceExtension`, `SymbolIconExtension` and `SymbolIconSourceExtension`.
```

Reply:
Wait, I'm confused, don't we have links to all of them in this line you highlighted? Or do you mean to copy these again in a separate section too? 🤔